src/main/resources/db/migration/V20170713134558340__create_item_sequence.sql

```diff
@@ -0,0 +1,24 @@
+DELIMITER //
+CREATE PROCEDURE CreateSequence (`name` VARCHAR(30), `start` INT, `inc` INT)
+  BEGIN
+    CREATE TABLE IF NOT EXISTS `_sequences`
+    (
+      `name` VARCHAR(70) NOT NULL UNIQUE,
+      `next` INT NOT NULL,
+      `inc` INT NOT NULL
+    );
+
+    INSERT INTO `_sequences` VALUES (name, start, inc);
+  END //
+DELIMITER;
+
+CALL CreateSequence('item_id_seq', 1, 1);
+
+DELIMITER //
+CREATE FUNCTION NextVal (`vname` VARCHAR(30))
+  RETURNS INT DETERMINISTIC
```

Comment:
I was confused, no doubt about that.  I had it all working locally and then AWS did not work.  I bumped across the replication stuff and for really no other reason than it worked, picked DETERMINISTIC.   I'll leave it as DETERMINISTIC.